chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -95,3 +81,31 @@ void BraveRenderViewContextMenu::AddSpellCheckServiceItem(
     bool is_checked) {
   // Suppress adding "Spellcheck->Ask Brave for suggestions" item.
 }
+
+void BraveRenderViewContextMenu::InitMenu() {
+  RenderViewContextMenu_Chromium::InitMenu();
+
+  // Add Open Link with Tor
+  int index = -1;
+  if (!params_.link_url.is_empty()) {
+    const Browser* browser = GetBrowser();
+    const bool is_app = browser && browser->is_app();
+
+    index = menu_model_.GetIndexOfCommandId(
```

Comment:
cc @petemill - anything else she should do here? Did you find a good way to test?